content/docs/code-splitting.md

```diff
@@ -40,86 +39,90 @@ function add(a, b) {
 console.log(add(16, 26)); // 42
 ```
 
-> Note:
+> Notatka:
```

Comment:
```suggestion
> Uwaga:
```

To jest nasze utarte tłumaczenie, które znajduje się w naszym glosariuszu na Slacku (do którego pewnie jeszcze nie masz dostępu ;-) ).